scripts/qwen2vl_qnn/push_eval_assets.sh

```
@@ -0,0 +1,44 @@
+  exit 1
+fi
+
+echo "ADB              : ${ADB_BIN}"
+echo "Cases file       : ${CASES_FILE}"
+echo "Local assets     : ${ASSET_SRC}"
+echo "Remote image dir : ${REMOTE_IMAGE_DIR}"
+
+"${ADB_BIN}" shell "mkdir -p $(remote_quote "${REMOTE_IMAGE_DIR}")" </dev/null
+
+declare -A pushed=()
+while IFS='|' read -r case_id image_file prompt; do
```

Shellcheck (0.11.0)

[warning] prompt appears unused. Verify use (or export if used externally).

(SC2034)